spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -235,4 +235,13 @@
       expect(current_path).to eq profile_path
     end
   end
+
+  describe 'clicking the logo image during 2fa process' do
+    it 'signs the user out and returns them to the home page' do
+      user = build_stubbed(:user, :signed_up)
+      sign_in_user(user)
+      find("img[alt='login.gov']").click
+      expect(current_path).to eq root_path
```

Comment:
However, to Jessie's point, if you also wanted to make sure that the user was signed out, I would use a controller spec because you can't really test that in a feature spec, afaik. Even if all we wanted to test was that partially signed in users could access the root url, I would still prefer a controller spec since it's faster than a feature spec.

The problem with only testing that the current path is the root path is that it is not a reliable way to determine whether the user has been signed out. For example, we could have code that redirects a signed-in user to the profile page, then signs them out via `sign_out`. In that case, the current path will still be the profile page, but the user will be signed out. 

Or, we might want to allow partially signed in users to access the root url without signing them out, in which case we also want to test that they are not signed out.